DecipherNow/gm-fabric-dashboard
# feat: Added copyright text and updated footer responsiveness behavior

Resolves #1315 
Resolves #1341

## Proposed Changes

  - Add additional copyright text to application footer
  - Rewrite responsive styling to unpin footer from viewport on small screens
  - Improve appearance of footer on small screens.
  
## How to Test

  1. View app on a screen with <800px viewport width.
  2. Ensure copyright text is complete and readable.
  3. View app on a screen with >800px viewport width.
  4. Ensure copyright text is complete and readable.
